dawn/src/dawn/Serialization/IIRSerializer.cpp

```diff
@@ -152,6 +154,12 @@ static void computeInitialDerivedInfo(const std::shared_ptr<iir::StencilInstanti
   }
 }
 
+static void checkAndUpdateMaxID(int newID, int& maxID) {
```

Comment:
Do we need a function here? Is this just
```
#include <cmath>
#include <algorithm>
maxID = std::max(maxID, std::abs(newID));
```

Reply:
do you want to have it as that one liner everywhere or do you want it wrapped in a function anyway?